recirq/otoc/loschmidt/tilted_square_lattice/analysis-walkthrough.ipynb

```diff
@@ -0,0 +1,373 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Loschmidt Analysis Walkthrough"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "import matplotlib\n",
+    "matplotlib.rcParams.update(**{\n",
+    "    'axes.titlesize': 14,\n",
+    "    'axes.labelsize': 14,\n",
+    "    'xtick.labelsize': 12,\n",
+    "    'ytick.labelsize': 12,\n",
+    "    'legend.fontsize': 12,\n",
+    "    'legend.title_fontsize': 12,\n",
+    "    'figure.figsize': (7, 5),\n",
+    "})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from recirq.otoc.loschmidt import TiltedSquareLatticeLoschmidtSpec\n",
+    "\n",
+    "assert TiltedSquareLatticeLoschmidtSpec, 'register deserializer'"
```

Comment:
Why would this not exist?

Reply:
tl;dr: I removed this cell.

There is a bit of tech debt I'd like to punt on for now. For json deserialization to work, classes need to register their `cirq_type` with the big list in Cirq. By convention, this registration logic happens in `__init__.py` files (for example in `cirq_google`'s `__init__.py`). Unlike in Cirq, we do not recursively import all submodules when you `import recirq` so if you don't import anything from `recirq.otoc.loschmidt` deserialization will fail for json files containing `TiltedSquareLatticeLoschmidtSpec`.

However! Since we import the `analysis` module, that will cause the json deserialization registration code to execute.